### ISSUE TITLE
Check mark to save note is confusing, better use a save symbol

### ISSUE THREAD

**Issue by chaoflow** [orgzly-revived/orgzly-android-revived#192]
Device: Pixel 7a
    Android Version: 14 
    Orgzly Revived Version: 1.8.17 (fdroid) 

I experienced myself tapping the check mark, top-right when editing/viewing a note in detail, to mark the note as done. I think it would be better to use a save icon, given also that we use a similar check mark in the agenda view to actually mark a note as done.

**Comment by amberin** [comment]
I agree, but isn't there a risk with the save icon that the user would expect to stay in the edit screen? I feel it's unusual for a "save" button to mean "save and exit", whereas the checkmark may mean "I'm done here".

**Comment by SeerLite** [comment]
The Tasks.org uses a save icon at the top left. I've never questioned whether or not it will save and exit, I always assumed it did. Maybe it has to do with the fact that it sits at the top left.

I also had this issue on Orgzly Revived, I wanted to save but didn't know if the checkmark would mark it as done instead.

**Comment by chaoflow** [comment]
> I agree, but isn't there a risk with the save icon that the user would expect to stay in the edit screen? I feel it's unusual for a "save" button to mean "save and exit", whereas the checkmark may mean "I'm done here".

Two thoughts:
1. Do we need "Save and exit" or could this be "Click Save", "Swipe back"?
2. How about replacing the checkmark with "Save and exit"?

**Comment by ShuoShen** [comment]
> The Tasks.org uses a save icon at the top left. I've never questioned whether or not it will save and exit, I always assumed it did.
> How about replacing the checkmark with "Save and exit"?

A "Save" button on mobile usually just saves and exits to previous view. An example of this is google calendar when the user saves the edits. So I think it's reasonable to drop "and exit" for brevity. 

@chaoflow, personally, the check mark conveys the same message as the "Save" button. However, I understand this is confusing to you and probably to many others, and it's better to have a button with better clarity.

**Comment by amberin** [comment]
I also just noticed that the label on the checkmark icon says "Done", which is very unclear, and probably has been translated to all kinds of things. :)

**Comment by Ojd2000** [comment]
I'm answering also in relation to the issue of mine mentioned by @amberin above.

Firstly, I found a fairly standard save-and-close icon by searching the web: a floppy disk with a check mark at the bottom-right corner. I think it's also intuitive enough to be used, given its similarity with the floppy-disk-only icon.

I also have an idea to avoid giving the user two different buttons to either save-only and save-and-close: keep the save-and-close button, with a new icon, and trigger a save-only action when the virtual keyboard is closed.

This would also require highlighting the cursor's last position in the note's content field.

**Comment by QJKX** [comment]
The [Material Design system for Dialogs](https://m3.material.io/components/dialogs/guidelines) version, is  an `X` close button in the top left (that causes a confirmation prompt if you changed stuff without saving), and a text label on the top right of `Save`, that saves and closes.

**Comment by amberin** [comment]
So how about we replace the ambiguous symbol with a "Save" text button, as in Google's examples  above?
